components/Componenten/_button/button.njk

```diff
@@ -4,6 +4,13 @@
   {%- if roleDescription %} aria-roledescription="{{ roleDescription }}"{% endif %}
   {%- if disabled %} disabled{% endif -%}>
   {%- if icon and not(iconOnly) %}{% render '@icon', {icon: icon} -%}{% endif -%}
-  <span {{ className([iconOnly, 'sr-only']) }}>{{ label }}</span>
+  {%- if not(iconOnly) %}<span>{{ label }}</span>{% endif %}
```

Comment:
We doen deze functionaliteit niet op button, graag reverten. Vergeet de dom reference files niet opnieuw in te checken